src/server/oasisapi/portfolios/viewsets.py

```diff
@@ -156,7 +161,8 @@ def storage_links(self, request, pk=None, version=None):
 
 
     @swagger_auto_schema(methods=['get'], responses={200: FILE_RESPONSE}, manual_parameters=[FILE_FORMAT_PARAM])
-    @action(methods=['get', 'delete'], detail=True)
+    @swagger_auto_schema(methods=['post'], manual_parameters=[FILE_VALIDATION_PARAM])
+    @action(methods=['get', 'post', 'delete'], detail=True)
     def accounts_file(self, request, pk=None, version=None):
         """
         get:
```

Comment:
missing docstring for `post` method

Reply:
Should be already fixed from https://github.com/OasisLMF/OasisPlatform/pull/724/commits/77e932044c47066bb2ce4a1dadb28f4592cffe46 

Or am i missing something?

Reply:
that's right..I don't know why it wasn't showing up...